External/OpenACCV_V/CMakeLists.txt

```diff
@@ -0,0 +1,70 @@
+# OpenACCV&V Validation & Verification Suite
+# https://github.com/OpenACCUserGroup/OpenACCV-V
+
+include(External)
+
+option(TEST_SUITE_FORCE_ALL "Execute all OpenACC V&V tests, even those known to be unsupported by Clang" OFF)
+
+set(TEST_SUITE_OFFLOADING_FLAGS --offload-arch=native CACHE STRING "Compiler arguments for offloading")
+set(TEST_SUITE_OFFLOADING_LDFLAGS --offload-arch=native CACHE STRING "Linker arguments for offloading")
+
+function (add_OpenACC_vv LANG)
+  if (NOT OpenACC_${LANG}_FOUND)
+    message(FATAL_ERROR "OpenACC for ${LANG} not found")
+    return ()
+  endif ()
+  if ("${LANG}" STREQUAL "C")
+    set(_langext ".c")
+  elseif ("${LANG}" STREQUAL "CXX")
+    set(_langext ".cpp")
+  elseif ("${LANG}" STREQUAL "Fortran")
+    set(_langext ".F90")
+  else ()
+    message(FATAL_ERROR "Unsupported languge ${LANG}")
+  endif ()
+
+  file(GLOB_RECURSE _tests_sources RELATIVE "${TEST_SUITE_OpenACCVV_ROOT}/Tests" "${TEST_SUITE_OpenACCVV_ROOT}/Tests/*${_langext}" )
+  foreach (_file IN LISTS _tests_sources)
+    get_filename_component(_ext "${_file}" EXT)
+    get_filename_component(_basename "${_file}" NAME_WE)
+    get_filename_component(_directory "${_file}" DIRECTORY)
+    string(REPLACE "." "" _ext "${_ext}")
+    set(_name "acctargetvv-${_basename}.${_ext}")
+
+    llvm_test_run()
+
+    llvm_test_executable(${_name} "${TEST_SUITE_OpenACCVV_ROOT}/Tests/${_file}")
+     
+    target_link_libraries(${_name} PUBLIC OpenACC::OpenACC_${_lang} m)
+
+    # Add -fopenacc to linker command line; for some reason this is not done by target_link_libraries.
+    target_link_options(${_name} PRIVATE ${OpenACC_${LANG}_FLAGS})
```

Comment:
So this comment isn't accurate, right?  It is adding the openacc-language flags, not the -fopenacc?  Or where are the `OpenACC_C_flags` and `OpenACC_CXX_flags` set?

Reply:
my fault - you were right before

Reply:
"OpenACC_C_flags "and "OpenACC_CXX_flags" are set in cmake. Check out the official FindOpenACC documentation at the following website: https://cmake.org/cmake/help/latest/module/FindOpenACC.html

But my error, currently find_package(OpenACC), only NVHPC, PGI, GNU and Cray compilers are supported, We cannot use this method to see if the current llvm supports OpenACC. "OpenACC_C_flags "and "OpenACC_CXX_flags" are currently invalid

Reply:
@wangxin0321 Is there a way around? Or we have to make an issue with cmake?